src/panels/lovelace/cards/hui-sensor-card.js

```diff
@@ -246,6 +242,7 @@ class HuiSensorCard extends EventsMixin(LitElement) {
           padding: 16px;
           position: relative;
           cursor: pointer;
+          --line-color: var(--accent-color, var(--primary-color));
```

Comment:
This is not used? I also don't think that we should fallback.

Reply:
You mean adding this here doesnt magically make it used... 🤦‍♂️